cluster/gce/util.sh

```diff
@@ -741,6 +741,47 @@ function create-master() {
   create-master-instance "${MASTER_RESERVED_IP}" &
 }
 
+function replicate-master() {
+  set-replica-name
+
+  local existing_master=$(gcloud compute instances list \
+    --project "${PROJECT}" \
+    --regexp "$(get-replica-name-regexp)" \
+    --format "value(name,zone)" | head -n1)
+  local existing_master_name=$(echo "${existing_master}" | cut -f1)
+  local existing_master_zone=$(echo "${existing_master}" | cut -f2)
+
+  echo "Replicating existing master ${existing_master_zone}/${existing_master_name} as ${ZONE}/${REPLICA_NAME}"
+
+  # Before we do anything else, we should configure etcd to expect more replicas.
+  gcloud compute ssh ${existing_master_name} \
+    --project "${PROJECT}" \
+    --zone "${existing_master_zone}" \
+    --command \
+      "curl localhost:4001/v2/members -XPOST -H \"Content-Type: application/json\" -d '{\"peerURLs\":[\"http://${REPLICA_NAME}:2380\"]}'; \
+       curl localhost:4002/v2/members -XPOST -H \"Content-Type: application/json\" -d '{\"peerURLs\":[\"http://${REPLICA_NAME}:2381\"]}'" 2>/dev/null
```

Comment:
We should probably check if the "event" etcd instance exists (I think there is a way to disable it).